docs/userGuide/syntax/searchBars.mbdf

```diff
@@ -33,6 +33,7 @@ Enter a search term (eg. 'search bar') to see the search result dropdown.
 
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
+algolia | `Boolean` | `false` | Whether the searchbar should be connected to Algolia Docsearch.
```

Comment:
Maybe link "Algolia Docsearch" to `{{ baseUrl }}/userGuide/usingPlugins.html#algolia-enabling-algolia-docsearch` as well?